hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirMkdirOp.java

```diff
@@ -222,7 +222,8 @@ private static INodesInPath unprotectedMkdir(FSDirectory fsd, long inodeId,
         timestamp);
 
     INodesInPath iip =
-        fsd.addLastINode(parent, dir, permission.getPermission(), true);
+        fsd.addLastINode(parent, dir,
+                permission.getPermission(), true, null, true);
```

Comment:
Keep the code style as usual, such as here, we should leave 4 space blanks compare to L225. Some other changes have the same format issues.

Reply:
ok